### PR TITLE
Feature/1234 change availability status from sidebar

### LINKED ISSUE #1234: Enhancement: Improve UX around availability status
### Problem

Right now, the option to change availability status is hidden in profile settings. People might not realize that they have an option offline. To improve this UX, provide a setting right on the sidebar to change the status

### Design

Screenshot 2020-09-14 at 5 24 24 PM

### PR DESCRIPTION
**Pull request by somenugget**
## Description
User is now able to change his availability status from a sidebar. 
status-change

I wasn't sure about how do you make translations for a project, so I have been using GTranslate.

Fixes #1234 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
- Log in
- See current availability status in the bottom of the sidebar
- Click "Change" button in the bottom of sidebar
- See three options in dropdown. One should be disabled
- Click on any available options
- See that status have changed in the bottom of the sidebar

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by pranavrajs** [comment]
@somenugget Thanks for the PR. This looks really nice. ❤️

File: app/javascript/dashboard/components/layout/Sidebar.vue [hunk 1]
```diff
@@ -55,11 +59,12 @@
           </ul>
         </div>
       </transition>
+
       <div class="current-user" @click.prevent="showOptions()">
         <thumbnail
           :src="currentUser.avatar_url"
           :username="currentUserAvailableName"
-          :status="currentUser.availability_status"
+          :status="null"
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/layout/Sidebar.vue:67]: Setting this not null doesn't make sense. I think we can remove this key altogether.

**Review by pranavrajs — DISMISSED** [review]
The code looks good, thanks for the test. I have a small comment around the usage of null. 

For translation, Google translate is fine. We have community translations enabled in https://translate.chatwoot.com, please would contribute the translations there. I made a small change for translation in my native language. 

We will test this PR and merge it.

**Comment by nithindavid** [comment]
@somenugget Great PR 🔥

File: app/javascript/dashboard/components/layout/Sidebar.vue (hunk 1 above)
Comment (somenugget) [reply app/javascript/dashboard/components/layout/Sidebar.vue]: Removed

**Comment by pranavrajs** [comment]
@somenugget I've tested this out. There is a bug introduced with the PR. 

In this PR, we are `updateProfile` action which has the following lines
https://github.com/chatwoot/chatwoot/blob/b862817b2959bc4c8e6f5dbd9ebaf2af2ad56a21/app/javascript/dashboard/api/auth.js#L121-L140

Once you change the status, displayName becomes empty. So I would suggest creating a separate action for availability status consisting of the `availability` only.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**